tailwind.config.js

```diff
@@ -18,6 +18,7 @@ module.exports = {
     },
     extend: {
       colors: {
+        idk: "#26241D",
```

Comment:
:(

Reply:
this was basically just to lightly match the brown from the figma we'll obviously have to do a styling pass and choose colors